### DISCUSSION TITLE
For PipeWire users, suggest EasyEffects as an alternative *for now* in the readme

### DISCUSSION THREAD

**Discussion by zoktiik** [noisetorch/NoiseTorch#301] [Ideas (I would suggest ...)]
While this project is sorting out its security issue and advising people not to use it, it would probably be a good idea to direct those users towards an alternative application, even if it is heavier and more complex than NoiseTorch. Giving people _an option_ other than NoiseTorch is better than _no option_, as they might choose to value convenience over a potential security risk.

**Comment by korelabs** [comment]
I've been using Cadmus in the meantime, not as awesome as noisetorch but it
seems to play a bit nicer in pipewire for me at least.

**Comment by TCMOREIRA** [comment]
Have the security issues been solved?

**Reply by AXDOOMER** [reply]
Yes. Download the latest version from the releases page.

**Reply by TCMOREIRA** [reply]
So good to hear that!!!
